src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/IDynamicInterfaceCastable.cs

```diff
@@ -56,7 +67,7 @@ public interface IDynamicInterfaceCastable
     /// <see cref="IDynamicInterfaceCastable" /> scenarios trimming friendly.
     /// </remarks>
     [AttributeUsage(AttributeTargets.Interface, AllowMultiple = false, Inherited = false)]
-    public sealed class DynamicInterfaceCastableImplementationAttribute : Attribute
+    public sealed partial class DynamicInterfaceCastableImplementationAttribute : Attribute
```

Comment:
```suggestion
    public sealed class DynamicInterfaceCastableImplementationAttribute : Attribute
```
Does this need to be partial?

Reply:
It does not. (this was from an earlier iteration where I put the static constructor here)